lua/omnisharp_extended.lua

```diff
@@ -196,25 +190,22 @@ M.handler = function(err, result, ctx, config)
       end
 
       local locations = M.definitions_to_locations(result.result.Definitions)
-      local handled = M.handle_locations(locations)
+      local handled = M.handle_locations(locations, client.offset_encoding)
       if not handled then
         return vim.lsp.handlers['textDocument/definition'](err, result, ctx, config)
       end
     end
   end
 
   local locations = M.textdocument_definition_to_locations(result)
-  local handled = M.handle_locations(locations)
+  local handled = M.handle_locations(locations, client.offset_encoding)
   if not handled then
     return vim.lsp.handlers['textDocument/definition'](err, result, ctx, config)
   end
 end
 
-M.handle_locations_telescope = function(locations, opts)
+M.handle_locations_telescope = function(locations, offset_encoding, opts)
   opts = opts or {}
-
-  local fetched = M.get_metadata(locations)
```

Comment:
This shouldn't be removed, the return value is unused, but `get_metadata` actually preloads the buffer when jumping into external assemblies